site/docs/contribute/modules.md

```diff
@@ -34,6 +34,9 @@ An example for Vault Read Secret API call which uses the Vault parameters is as
 $ curl --header "X-Vault-Token: ..." -X GET https://<address>/<secretPath>
 ```
 
+Fybrik repository contains a [Python Vault package](https://github.com/fybrik/fybrik/tree/master/python/vault) that modules can use to
+retrieve the credentials.
+
```

Comment:
and golang package [Golang Vault package](https://github.com/fybrik/fybrik/tree/master/pkg/vault)